### PR TITLE
secretmanager: update v1beta1->v1

### PR DESCRIPTION
**Pull request by gwhitehawk**
Client lib has been updated: https://pkg.go.dev/cloud.google.com/go@v0.55.0/secretmanager/apiv1

### CODE REVIEW

File: secretmanager/iam_revoke_access.go [hunk 1]
```diff
@@ -36,15 +37,19 @@ func iamRevokeAccess(w io.Writer, name, member string) error {
...
 	if err != nil {
 		return fmt.Errorf("failed to get policy: %v", err)
 	}
 
 	// Grant the member access permissions.
 	policy.Remove(member, "roles/secretmanager.secretAccessor")
-	if err = handle.SetPolicy(ctx, policy); err != nil {
+	setReq := &iampb.SetIamPolicyRequest{
+		Resource: name,
+		Policy: policy,
```
[the remark this answers is not in the document]
Comment (broady) [reply secretmanager/iam_revoke_access.go]: https://github.com/googleapis/google-cloud-go/issues/1882

[the remark this answers is not in the document]
Comment (gwhitehawk) [reply secretmanager/iam_revoke_access.go]: Oh sorry, I wondered why it no longer works, thought there were changes in IAM in the meantime

[the remark this answers is not in the document]
Comment (gwhitehawk) [reply secretmanager/iam_revoke_access.go]: @sethvargo Why is there still an import error in checks (form IAM method in client): https://github.com/GoogleCloudPlatform/golang-samples/pull/1307/checks? Where do I specify the secretmanager client version?

**Comment by gwhitehawk** [comment]
Thanks! Finally, sorry for many unsuccessful attempts.

**Pull request merged**